apps/ledger-live-desktop/src/renderer/components/DeviceAction/rendering.tsx

```diff
@@ -663,7 +664,15 @@ export const DeviceNotOnboardedErrorComponent = withV3StyleProvider(
       setTrackingSource("device action open onboarding button");
       dispatch(closeAllModal());
       setDrawer(undefined);
-      history.push(device?.modelId === "stax" ? "/sync-onboarding/manual" : "/onboarding");
+      if (!device?.modelId) {
+        history.push("/onboarding");
+      } else {
+        history.push(
+          DevicesWithTouchScreen.includes(device.modelId)
```

Comment:
[Ask] Why not use `isSyncOnboardingSupported`, because one day sync onboarding could be accessible for device like LNSP or LNX